src/bindings/c/src/CMakeLists.txt

```diff
@@ -38,20 +37,23 @@ set_target_properties(${TARGET_NAME} PROPERTIES
     )
 
 export(TARGETS ${TARGET_NAME} NAMESPACE openvino::
-       APPEND FILE "${CMAKE_BINARY_DIR}/OpenVINOTargets.cmake")
+    APPEND FILE "${CMAKE_BINARY_DIR}/OpenVINOTargets.cmake")
 
 # install
-
 ie_cpack_add_component(${OV_CPACK_COMP_CORE_C} DEPENDS core)
 ie_cpack_add_component(${OV_CPACK_COMP_CORE_C_DEV} DEPENDS ${OV_CPACK_COMP_CORE_C})
 
 install(TARGETS ${TARGET_NAME} EXPORT OpenVINOTargets
-        RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
-        NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_C_DEV}
-        INCLUDES DESTINATION ${OV_CPACK_INCLUDEDIR}/ie)
-
-install(DIRECTORY ${InferenceEngine_C_API_SOURCE_DIR}/include/
-        DESTINATION ${OV_CPACK_INCLUDEDIR}/ie
-        COMPONENT ${OV_CPACK_COMP_CORE_C_DEV})
+    RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT ${OV_CPACK_COMP_CORE_C}
+    NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_C_DEV}
+    INCLUDES DESTINATION ${OV_CPACK_INCLUDEDIR}/openvino/c)
+
+install(DIRECTORY ${OpenVINO_C_API_SOURCE_DIR}/include/c_api
+    DESTINATION ${OV_CPACK_INCLUDEDIR}/ie
+    COMPONENT ${OV_CPACK_COMP_CORE_C_DEV})
+
+install(DIRECTORY ${OpenVINO_C_API_SOURCE_DIR}/include/openvino
+    DESTINATION ${OV_CPACK_INCLUDEDIR}/openvino/c
```

Comment:
Why `openvino/c`?

Reply:
If will not avoid mixing  C++ header file and C header file

$tree include/openvino/ -L 1
include/openvino/
├── c
├── core
├── frontend
├── op
├── openvino.hpp
├── opsets
├── pass
└── runtime

Reply:
Ok, I understood thanks.

Reply:
If I understand correctly in final package we will have a path for C headers: `openvino/c/openvino.h`. Can we use the same path in our repository for new API?

Reply:
we use "openvino/openvino.h" for new C header, so we should get this path: openvino/c/openvino/openvino.h, maybe it is confusing, so should we change  to openvino/c/openvino.h and user use "openvino/c/openvino.h"?

Reply:
Yes, if we put C API near the C++ will be better to have "openvino/c/openvino.h. From my point of view

Reply:
Got it!

Reply:
It seems that we cannot apply such path(openvino/c/openvino.h) for package build and sample build:
if samples/c/hello_classification_c   use "openvino/c/openvino.h" , we must change the C header file to be:  "src/bindings/c/include/openvino/c/openvino.h", it is weird

So I  think "openvino/openvino.h" is ok, and install directory will be: include/openvino/c/openvino/openvino.h
or we use "openvino.h" in application replace "openvino/openvino.h".

Which one is better?
1)  source code path: src/bindings/c/include/openvino/openvino.h
     install path: include/openvino/c/openvino/openvino.h
     app include path: include/openvino/c
     app header file: #include "openvino/openvio.h"

2)  source code path: src/bindings/c/include/openvino/openvino.h
     install path: include/openvino/c/openvino.h
     app include path: include/openvino/c
     app header file: #include "openvino.h"

Reply:
Ideally I would like to have something like that:

- src/bindings/c/include/openvino/c/openvino.h
- inside openvino usage: "openvino/c/openvino.h"
- inside application usage: "openvino/c/openvino.h"

But in this case we have a mix between C and C++ API as I mentioned today in previous comment. Today I will think about it, because I want to align paths for openvino and customer application and at the same moment I don't want to have a mix of c and C++ APIs.

Maybe we can use include path something like: `c_openvino/openvino.h` but I am not sure about this, because I dislike `c_` prefix.

Reply:
done